docs/dxw-support-runbook.md

```diff
@@ -0,0 +1,75 @@
+# dxw support runbook
+
+The audience of this document is a developer working on dxw first-line support.
+It explains how to perform some tasks that you might get asked to do. It assumes
+that you have not worked on this service before.
+
+## I want to make a bug fix and deploy it
+
+### You will need
+
+- permissions to open a pull request in the repository
+- credentials for the project’s Azure infrastructure
```

Comment:
Maybe. This might make sense to a DfE support team, but I'm not sure a random developer in dxw would know what this means. I'll add a way for them to confirm this, like "if you are not sure whether you have this: if you are able to log in to <this site> and view <this resource>, you probably have it".